libraries/aws_vpn_connections.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsVpnConnections < AwsResourceBase
+  name 'aws_vpn_connections'
+  desc 'Verifies settings for an AWS VPN connections in bulk'
+
+  example '
+    describe aws_vpn_connections do
+      it { should exist }
+    end
+  '
+  attr_reader :table
+
+  # FilterTable setup
+  FilterTable.create
+             .register_column(:vpn_connection_id,  field: :vpn_connection_id)
+             .register_column(:vpn_gateway_id,     field: :vpn_gateway_id)
+             .register_column(:tunnel_options,     field: :tunnel_options)
+             .register_column(:state,              field: :state)
+             .register_column(:type,               field: :type)
+             .register_column(:tags,               field: :tags)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    vpn_connection_rows = []
+    catch_aws_errors do
+      @vpn_connections = @aws.compute_client.describe_vpn_connections.to_h[:vpn_connections]
```

Comment:
The `describe_vpn_connections` method returns an array of vpn connection [object](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/EC2/Types/VpnConnection.html). You might consider removing the `to_h` method and looping through the `@vpn_connections.vpn_connections` array. Then accessing each attribute via dot notation, e.g: `vpn_connection.vpn_connection_id` instead of `vpn_connection[:vpn_connection_id]`. If you follow that route, then renaming the `@vpn_connections` to `@api_response` might make more sense.

If it is working fine as it is, you can ignore my suggestion.